### ISSUE TITLE
Add a text searchable keybinding reference

### ISSUE THREAD

**Issue by tukoz** [The-Compiler/qutebrowser#1330]
A Firefox / Xombrero / Palemoon Pentadactyl user, it took me over an hour to find most (not all) basic commands and key bindings to begin testing qutebrowser (mostly thanks to the dwb list showed in Refs below). May be my own eye's fault! 

I suggest adding in good eye site a *text* searchable qutebrowser's keybindings list beside the present non-searchable [image ref](http://qutebrowser.org/img/cheatsheet-big.png). I wrote a fu***ingly limited table as an example:

Action 				| Xombrero 		| Pentadactyl	 | dwb	 	| qutezilla 	
------------------------------- | --------------------- | -------------- |-------------	| ------------  
**Commands**			| 			|                |		|
Last cmd(s)			| `:↑`			| `:↑`         	 | `.`		| `:` Ctrl-P/N	
Add/edit cmd			| conf file (limited)	| 	         |		| ``alias = cmd` in conf file
				| 			|                |		|
**Open**			| `:o[pen]`		| `:o`         	 | o		| o, `:open`		
Open in new tab			| `:tabnew`, F11 	|                | O 		| O, :`open -t`	
Open hyperlink			| f-[N]		        | f-[N]      	 |  		| f-[N]
 " " in new tab			| F-[N]		        | F-[N]	         |    		| F-[N]
 " " in new bg tab		|     			|     		 | xo		| xo, `:open -b`
Next nav action open in new tab	| 			|                | V 		|
				| 			|                |		|
**History**			| 			|                |    		|
Go back/forward 		| Alt-←|→, `:goforw/back` | A-←|→    | L   	        | L|`:forward`
Show history			| Alt-h		        |                | Sh		| ?		
List sessions			| `:session list`	|                |		|
Load last default session	| 			|                |    		|
Load session X			| 			|                |		|
				| 			|                |		|
**Tabs**			| 			|                |    		|
Go to next/prev	tab		| Ctrl-←|→		|                |    		| ?|Shift-Tab
Go to tab N			| Ctrl-[N]→		| Alt-[N]      |    		| ?
Switch to last	opened tab	|			|	         |    		| Ctrl-Tab (works weird)
				| 			|                |		|
**Search**			| 			|                |    		|
Search with da SE		| 			|                |    		|
Add SE to search		| conf file		| ?              | gs		| ?
Search item in page		| `/<item>`		| `/<item>`      |		| `/<item>`
Search item next occurrence	| n			| `n`            |		| `n`
				| 			|                |		|
**URLs**			| 			|                |    		|
Display link url		| on mouse hover	|                |		| ?		
Copy url			| y (clic url in statusbar) 	| `:pa[geinfo]`	 |		| yy
Page has feed?			| ?			| `:pa`, C-g, gC-g |   		| ?
				| 			|                |		|
**Features**			| 			|                |    		|
Javascript & 			| Ctrl-j		| NoScript       | ttu	| ?
Cookies toggle,			| `:cookie toggle`	| `:set cookie`.. | CT 	| 
 whitelist			| `:js|cookie save`, conf |		 | tsh|CS	| ?

### Refs
- [Pacman/Rosetta](https://wiki.archlinux.org/index.php/Pacman/Rosetta)
- [Xombrero keybindings](https://opensource.conformal.com/wiki/xombrero#Browsing)
- [Pentadactyl keybindings](http://5digits.org/help/map.xhtml)
- [dwb keybindings](http://portix.bitbucket.org/dwb/resources/dwb.1.html)  
  Useful dwb keybindings #13
- qutebrowser: Show received keystrokes #658
- qutebrowser: Edit keybindings #781

**Comment by lahwaacz** [comment]
The full text-searchable list of qutebrowser's keybindings can be found (and configured) at `~/.config/qutebrowser/keys.conf`. Reference of commands is available at `qute://help/commands.html` (accessible by `:help`).

**Comment by tukoz** [comment]
Yeap that's what The-Compiler sayed on #qutebrowser (I started to edit it according to my fingers habits). And `keys.conf` is thematised + well annotated which helps.

Now for the potential tester, which easy way exists that allows him or her to figure out which browser/editor's keybinding qutebrowser is the closest?

A qutebrowser rosetta like [Pacman's one](https://wiki.archlinux.org/index.php/Pacman/Rosetta) would be of considerable help/time saver.

Also once he installs it there's a quite a lot of help available, but I for example haven't seen `keys.conf` mentionned in it.

So after time spent searching and questioning the qutebrowser tester opens `keys.conf` and search which commands/keystrokes opens URL in a new page, display history, and so on. Is he realy supposed to find that in this file?

```
set-cmd-text -s :open -t 	# that's Open in new tab dude!
    O
```

Now he/she whishes to know how to go to tab N or move a tab to N.
keys.conf:
```
tab-move
    gm

tab-move -
    gl

tab-move +
    gr
```
qute://help/commands.html:
```
Syntax: :tab-move [direction]

Move the current tab.

positional arguments

direction: + or - for relative moving, not given for absolute moving.

count

If moving absolutely: New position (default: 0) If moving relatively: Offset.

tab-next
Switch to the next tab, or switch [count] tabs forward.

count

How many tabs to switch forward.

tab-only
```
:gniee?:  
`:tab-move 5, :tab-move5, :tab-move=5 and :5tab-move` I tried them all and failed all before The-Compiler told me it's `:N:cmd` (of couuuurse!).

I mean both qutebrowser developers and new users have potentialy more enjoyable things to do than asking/answering how to go to tab N, don't we?

So I may write such a rosetta as these two files really do not address the issue here.
